templates/slide.html

```diff
@@ -435,47 +435,32 @@ <h2 class="popover__title">{{disp_str[i]}}</h2>
  function onPrevPage() {
    // empty previous child elements of textLayer
    $("#textLayerid").empty();
-   course = "{{course_name}}";
-    lec = "{{lec_name}}";
    if (pageNum <= 1) {
      return;
    }
-  //Added updated_slide to keep track of slide name
    pageNum--;
    new_slide_name= "{{course_name}}".concat("---").concat("{{lec_name}}").concat("---slide").concat((pageNum-1).toString());
-   logdata = JSON.stringify({
-        action: 'prev',
-        updated_slide: new_slide_name, 
-        route: window.location.pathname
-    });
-    navigator.sendBeacon('{{base_url}}log_action', logdata);
+   log_action(new_slide_name, 'prev', window.location.pathname, '{{base_url}}');
    queueRenderPage(pageNum);
    changedSlide(pageNum, pageNum+1);
  }
  document.getElementById('prev').addEventListener('click', onPrevPage);
 
  function onNextPage() {
    // if current pageNum requested is not valid, then return
-   // the following two lines is not required as the next will be disabled\
-    course = "{{course_name}}";
-    lec = "{{lec_name}}";
-    new_slide_name = course.concat("---").concat(lec).concat("---slide").concat((pageNum).toString());
-    console.log(new_slide_name);
-    logydata = JSON.stringify({
-        action: 'next',
-        //Added updated_slide to keep track of slide name
-        updated_slide: new_slide_name, 
-        route: window.location.pathname
-    });
-    navigator.sendBeacon('{{base_url}}log_action', logydata);
-
-   if (pageNum >= pdfDoc.numPages) {
-     return;
+         
+     if (pageNum >= pdfDoc.numPages) {
+      return;
    }
+  
+   pageNum++;
+    new_slide_name= "{{course_name}}".concat("---").concat("{{lec_name}}").concat("---slide").concat((pageNum-1).toString());
+    log_action(new_slide_name, 'next', window.location.pathname, '{{base_url}}');
+   
    
    // empty previous child elements of textLayer
-   $("#textLayerid").empty();
```

Comment:
@Sarnabh02 This statement is missing in the new code. Please add it back in the same order.